libkbfs/backpressure_disk_limiter.go

```diff
@@ -185,6 +189,11 @@ func (bt *backpressureTracker) onBlocksDelete(blockResources int64) {
 	bt.updateSemaphoreMax()
 }
 
+func (bt *backpressureTracker) beforeDiskBlockCachePut(ctx context.Context,
```

Comment:
Yeah, but the `err` return is only used by comparing to `nil`, so if you make this not return an error, you can omit that case in the caller below.

`ctx` is passed through all methods of `backpressureDiskLimiter` (for logging), but not `backpressureTracker` (only in `beforeBlockPut`, where it's needed.)